viva_las_vegas_script_tests.py

```diff
@@ -462,6 +462,133 @@ def test_sending_request_and_rejecting_it_without_permission(self):
         self.switch_channel('leave-coordination')
         self._reject_request(username=self.test_username)
 
+    def test_sending_ill_request_without_working_from_home_and_rejecting_it(self):
+        """Tests if it's possible to send an ill request without working home. """
+
+        self.choose_general_channel()
+
+        self.send_message(
+            '{} болею'.format(self._bot_name)
+        )
+        assert self.check_latest_response_with_retries(
+            'Очень жаль. Ты в состоянии работать из дома в эти дни?\n'
+            'Да\n'
+            'Нет'
+        )
+
+        self.send_message(
+            '{} Болею и не работаю'.format(self._bot_name)
+        )
+        assert self.check_latest_response_with_retries(
+            'Я понял. Согласовано ли отсутствие с руководителем/тимлидом?\n'
+            'Да\n'
+            'Нет'
+        )
+
+        self.send_message(
+            '{} Нет, они не предупреждены, что я болею'.format(self._bot_name)
+        )
+        assert self.check_latest_response_with_retries(
+            'Тогда сначала предупреди, а потом вернись и повтори все снова!'
+        )
+
+    def test_sending_ill_request_with_working_from_home_and_rejecting_it(self):
+        """Tests if it's possible to send an ill request with working home. """
```

Comment:
Tests if it's not possible to send a partially completed ill request (with working from home).